util/unittests/test_html.py

```diff
@@ -345,7 +345,7 @@ def test_01_singledicttabledata(self):
                          'Testing single_dict_table_data() '
                          'with a string input.')
 
-    def test_02_singledicttabledata(self):
+    def test_singledicttabledata(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_singledicttabledata' from line 335

<!-- Comment by [SideCI](https://sideci.com) -->